### PR TITLE
Add RWR plus card extension page

### PR DESCRIPTION
**Pull request by mmoura-nc**
New page with the description of what is needed to start the process of extending Visa card.

### CODE REVIEW

File: common/extend-visa.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# What I need to extend my Visa?
```
Comment (Tschis) [inline common/extend-visa.md]: typo: "What do I need"

File: common/extend-visa.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Rot-Weiß-Rot Karte (Red-White-Red Card)
+**TBD**
+
+## Rot-Weiß-Rot plus Karte (Red-White-Red plus Card)
+To initiate the process of extending the RWR plus card you need to proceed to the [MA 35](https://www.wien.gv.at/kontakte/ma35/) office that covers the district your are currently living. It is recommended that you start this process at least 3 months before your currently Visa date expiration.
```
Comment (Tschis) [inline common/extend-visa.md]: typos: 
- "**you are** currently living **in**"
- "your current visa expires" or "your current visa expiration date"

File: common/extend-visa.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+You will need to present the follow documentation, to assure that you are in order with their imigration laws:
```
Comment (Tschis) [inline common/extend-visa.md]: typo: "**following** documentation"

File: common/extend-visa.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+- KSV1870¹
+- Application Form²
+- Passport photo no older than 6 months
+- Copies of: 
+	Passport
+	Current Visa
+	Rent Contract
+	Last rent payment
+	Work contract (if you are currently working)
+    3 lastest payment slips (yours, if you are currenly working)
```
Comment (Tschis) [inline common/extend-visa.md]: typo: "latest"

File: common/extend-visa.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+- Meldezettel (if you move since your last application)
```
Comment (Tschis) [inline common/extend-visa.md]: typo: "moved"

File: common/extend-visa.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+ If your RWR card plus it based on your patner Visa, and this person it the money provider, it is import to also have the copies of his/her documents:
```
Comment (Tschis) [inline common/extend-visa.md]: typos:
- "is based"
- "partner"
- "is the money provider"
- "it is important"

File: common/extend-visa.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+ - KSV1870
+ - Passport
+ - Visa
+ - Work contract
+ - 3 lastest payment slips
```
Comment (Tschis) [inline common/extend-visa.md]: typo: "latest"

File: common/extend-visa.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+
+After you present the documentation to the MA35 representant, they will scan your finger digitals and hand you a payment bill for the new card that currently costs 160,00.
```
Comment (Tschis) [inline common/extend-visa.md]: Suggestion: add date to "currently", also add "euros" or "€"

File: common/extend-visa.md [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+
+¹If everything goes well, it takes around 30 to 40 days for new RWR plus card be ready. You need to comeback to the same office with the pay bill receipt and some valid Id documentation (Passaport or currently Visa).
```
Comment (Tschis) [inline common/extend-visa.md]: typos: 
- "for the new", 
- "card to be ready"
- "come back"
- "paid bill"
- "passport"
- "current visa"

File: common/extend-visa.md [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+
+²The [KSV1870](https://www.ksv.at/en) is a report that confirms to the authorities or your landlord that you’ll be financially reliable. It is valid for 3 months and currently cost about 30,00.
```
Comment (Tschis) [inline common/extend-visa.md]: typo: "costs"

Also, add "euros" or "€"

File: common/extend-visa.md [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+
+The Application Form is not currently available on the Internet, you will most probable find in the MA 35 office.
```
Comment (Tschis) [inline common/extend-visa.md]: typo: 
- "most probably"
- "find it in"

**Review by Tschis — CHANGES_REQUESTED** [review]

File: common/extend-visa.md (hunk 6 above)
Comment (Tschis) [reply common/extend-visa.md]: > typos:
> 
> * "is based"
> * "partner"
> * "is the money provider"
> * "it is important"

My bad:  "partner's"

**Comment by Tschis** [comment]
LGTM

File: common/extend-visa.md [hunk 12]
```diff
@@ -0,0 +1,39 @@
...
+- Meldezettel (if you moved since your last application)
+
+ If your RWR card plus is based on your partner's Visa, and this person is the money provider, it is important to also have the copies of his/her documents:
+ - KSV1870
+ - Passport
+ - Visa
+ - Work contract
+ - 3 latest payment slips
+
+After you present the documentation to the MA35 representant, they will scan your finger digitals and hand you a payment bill for the new card that currently (10/2019) costs 160,00 €.
```
Comment (brunojppb) [inline common/extend-visa.md]: I believe you meant `fingerprints`

Comment (mmoura-nc) [reply common/extend-visa.md]: Yes, thank you.

**Pull request merged**